numbat/modules/units/misc.nbt

```diff
@@ -30,6 +30,16 @@ unit calorie: Energy = 4.184 joule
 @aliases(Btu)
 unit BTU: Energy = 1055.05585262 joule
 
+@name("Therm")
+@url("https://en.wikipedia.org/wiki/Therm")
+@aliases(therms, thm : short)
+unit therm: Energy = 1.05506e8 joule
```

Comment:
In #471, you wrote that there are multiple definitions. It would be good to document which one we chose. You can use the `@description(…)` decorator for that (it's not used for other units yet since this is a relatively recent feature).

Also, it would be great if we could define `therm` in terms of BTU. For the thest above, we would leave the value in Joule, but here, I think we could define
```suggestion
unit therm: Energy = 10_000 BTU
```
if that matches the definition.

Reply:
> I would suggest to create a new issue with title something like this "Use @description(...) decorator to add descriptions of units with multiple definitions

Let's please do that while we're at it anyway. Now we still know what kind of definition we picked. Opening a new ticket to do that for other units as well sounds like a good idea though.

Reply:
> Using `BTU` instead of `joule` will remove the need to add description as the definition of `therm` depends on the definition of `BTU`.

Hm, not really. Users will not look up the definition of the unit in the source code. They will look that up in the documentation or via `info BTU`. `info BTU` includes the description field already, and we should probably add that to the [docs](https://numbat.dev/doc/list-units.html) as well.

Reply:
> Users will not look up the definition of the unit in the source code. They will look that up in the documentation or via `info BTU`

I guess I was not able to convey my point. `1 therm` is exactly equal to `100_000 BTU`. `info therm` will show user this value `1 therm = 100_000 BTU`. Now the definition of `BTU` used in prelude will determine the value of `therm` in `joules`. `info BTU` should show user which exact definition of `BTU` is being used. And description is currently missing in the documentation for `BTU`

Let me know if I am misunderstanding something. Thank you : )

Reply:
> `1 therm` is exactly equal to `100_000 BTU`. `info therm` will show user this value `1 therm = 100_000 BTU`.

Sure. But even if it's defined in terms of BTU, it still makes sense to write down which version of `therm` this is in a simple textual description for the user. If we don't add one, the user might not even know that there are multiple definitions... and that they might have chosen the wrong one for their use case.

Anyway, I'll merge this as-is. Maybe someone else feels like doing that in the future.

Reply:
> If we don't add one, the user might not even know that there are multiple definitions... and that they might have chosen the wrong one for their use case.

this is a valid point, adding a description for the unit `therm`